src/System.Private.CoreLib/shared/System/Collections/Generic/ArraySortHelper.cs

```diff
@@ -49,7 +49,7 @@ internal partial class ArraySortHelper<T>
     {
         #region IArraySortHelper<T> Members
 
-        public void Sort(T[] keys, int index, int length, IComparer<T>? comparer)
+        public void Sort(Span<T> keys, int index, int length, IComparer<T>? comparer)
```

Comment:
index and length are not necessary. The caller can make the right slice of the Span instead.